bundles/extensions/i18n/src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -226,6 +257,14 @@ public void handleEvent(final org.osgi.service.event.Event event) {
         }
     }
 
+    private void discardBundlesWithLocalePrefix(String localePrefix) {
```

Comment:
I was refering to basename and you are only referring to locales. We are talking about different things here. basename is orthogonal to locales.